manifests/config.pp

```diff
@@ -74,6 +75,19 @@
     }
   )
 
+  # The --no-logger flag that disables syslog output must either be present or
+  # not at all (there is no --logger) flag. This means a mesos::property can't
+  # be used.
```

Comment:
The way that `mesos::property` works (and, by extension, all the other Marathon config parameters) is that for boolean parameter a file is created for:
* `true` => `?parameter` which maps to a CLI flag `--parameter`
* `false` => `?no-parameter` which maps to a CLI flag `--no-parameter`

What I was trying to say is that the `--no-logger` flag is an exception to this. There is no `--logger` flag and a file called `?logger` would result in an error.